x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/api.ts

```diff
@@ -99,18 +100,15 @@ export const fetchRules = async ({
  * @param id Rule ID's (not rule_id)
  */
 export const fetchRuleById = async ({ id, signal }: FetchRuleProps): Promise<Rule> => {
-  const response = await fetch(`${chrome.getBasePath()}${DETECTION_ENGINE_RULES_URL}?id=${id}`, {
+  const response = await npStart.core.http.fetch<Rule>(`${DETECTION_ENGINE_RULES_URL}?id=${id}`, {
```

Comment:
`404` here as well, same encoding issue with the `?`.

Request URL from dev tools:
`http://localhost:5601/api/detection_engine/rules%3Fid=2190a33d-4972-4471-a1aa-7f301c8faefb`